calyx-ir/src/control.rs

```diff
@@ -319,19 +342,6 @@ pub enum StaticControl {
     Invoke(StaticInvoke),
 }
 
-impl Control {
-    pub fn is_static(&self) -> bool {
-        matches!(self, Control::Static(_))
-    }
-
-    pub fn get_latency(&self) -> Option<StaticLatency> {
-        match self {
-            Control::Static(sc) => Some(sc.get_latency()),
-            _ => None,
-        }
-    }
-}
-
```

Comment:
For some reason there was `impl Control` twice in the same file. I decided to change this bc it was confusing.